docs/features/pdb-compilation-options.md

```diff
@@ -257,20 +255,28 @@ Example:
 * verbose
 * warnaserror
 
-#### Options For CSharp
-
-See [compiler options](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/listed-alphabetically) documentation
+#### Shared Options for CSharp and Visual Basic
 
 | PDB Key                | Format                                  | Default   | Description  |
 | ---------------------- | --------------------------------------- | --------- | ------------ |
-| language               | `CSharp`                                | required  | Language name. |
-| language-version       | `[0-9]+(\.[0-9]+)?`                     | required  | [langversion](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/langversion-compiler-option) |
+| language               | `CSharp|Visual Basic`                   | required  | Language name. |
```

Comment:
Seems like markdown interprets `|` as table column delimiter even when enclosed in backticks :-/

Reply:
Maybe escaping would help `\|`?

Reply:
I think this should fix it:

```suggestion
| language               | `CSharp\|Visual Basic`                   | required  | Language name. |
```

Reply:
Yeah the `\` escaping fixed it.